__tests__/common/constants/__snapshots__/SpecialCharacter.test.js.snap

```diff
@@ -2,6 +2,7 @@
 
 exports[`Constants: PERMISSION Matches snapshot 1`] = `
 Object {
+  "BLACK_CIRCLE": "●",
```

Comment:
I don't know what this is - it was failing a test and I updated the snapshot thingy, I assume it was added some time ago?

Reply:
Um yeah meh either. Need to remove the snapshots..